tests/0089-max_poll_interval.c

```diff
@@ -430,17 +440,67 @@ do_test_rejoin_after_interval_expire(rd_bool_t forward_to_another_q) {
 
         if (forward_to_another_q)
                 rd_kafka_queue_destroy(polling_queue);
+        if (forward_to_consumer_q)
+                rd_kafka_queue_destroy(forwarder_queue);
         rd_kafka_queue_destroy(consumer_queue);
         test_consumer_close(rk);
         rd_kafka_destroy(rk);
 
         SUB_TEST_PASS();
 }
 
+static void consume_cb(rd_kafka_message_t *rkmessage, void *opaque) {
+        TEST_SAY("Consume callback\n");
+}
+
+static void do_test_max_poll_reset_with_consumer_cb(void) {
```

Comment:
Documentation comment above this test
We can also mention issue # for the original case

Reply:
Thanks, added.